src/Components/Modal/Modal.tsx

```diff
@@ -143,4 +211,28 @@ export const ModalOverlay = styled.div`
   background: rgba(200, 200, 200, 0.5);
 `
 
+export const CloseButton = styled(Icon).attrs({
+  color: Colors.graySemibold,
```

Comment:
Instead of using the colors here, I recommend grabbing them from palette.

https://github.com/artsy/palette/blob/master/src/Theme.tsx#L37

You can use it by...

```typescript
import { color } from "@artsy/palette"

color("black60");
```

If your interested in why the colors are named the way that they are, checkout the design system doc in notion. 

https://www.notion.so/artsy/Color-a0c24896daf8433d9409aee2146ac267